central/views/imagecveflat/db_response.go

```diff
@@ -0,0 +1,70 @@
+package imagecveflat
+
+import (
+	"time"
+
+	"github.com/stackrox/rox/generated/storage"
+)
+
+type imageCVEFlatResponse struct {
```

Comment:
Why it's not a storage proto?

Reply:
Because it is getting pieces of data.  This is a way for us to use the search framework to pull data out in a shape that differs from what is in the storage protos.  In this case we can have multiple storage.ImageCVEV2 instances and we want to get what amounts to an aggregated view of denormalized data.  Does that help?